packages/viewer/src/domain/Schema.tsx

```diff
@@ -15,27 +15,21 @@ export const Schema: FC<{
 }> = ({ heads }) => {
   return (
     <TableComponent>
-      <Thead>
-        <Tr>
-          <Th />
-          {heads.map(({ id }) => (
-            <Th key={id}>{id}</Th>
-          ))}
-        </Tr>
-      </Thead>
       <Tbody>
-        <Tr>
-          <Th>Type</Th>
-          {heads.map(({ id, type }) => (
-            <Td key={id}>{type}</Td>
-          ))}
-        </Tr>
-        <Tr>
-          <Th>Mode</Th>
-          {heads.map(({ id, mode }) => (
-            <Td key={id}>{mode}</Td>
-          ))}
-        </Tr>
+        <Thead>
+          <Tr>
+            <Tr>
+              <Th>Column</Th>
+              <Th>Type</Th>
+              <Th>Mode</Th>
+            </Tr>
+            {heads.map(({ id, type, mode }) => (
+              <Tr><Th key={id}>{id}</Th>
+              <Th key={id}>{type}</Th>
+              <Th key={id}>{mode}</Th></Tr>
+            ))}
+          </Tr>
+        </Thead>
```

Comment:
A `THead` is set for a child of `TBody`; the `THead` and `TBody` must be placed in the same hierarchy as children of `Table`:

```jsx
<Tbody>
  <Thead>
  </Thead>
  <TBody>
  </Tbody>
</Table>
```

See [\<table\>: The Table element \- HTML: HyperText Markup Language \| MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table).